lib/livebook/runtime/evaluator.ex

```diff
@@ -620,6 +620,60 @@ defmodule Livebook.Runtime.Evaluator do
     end
   end
 
+  defp eval("erlang", code, binding, env) do
+    try do
+      erl_binding =
+        binding
+        |> Enum.reduce(%{}, fn {name, value}, erl_binding ->
+          :erl_eval.add_binding(
+            name
+            |> :erlang.atom_to_binary()
+            |> Macro.camelize()
+            |> :erlang.binary_to_atom(),
+            value,
+            erl_binding
+          )
+        end)
+
+      {:ok, tokens, _} = :erl_scan.string(code |> String.to_charlist())
+      {:ok, parsed} = :erl_parse.parse_exprs(tokens)
+
+      {:value, result, new_erl_binding} =
+        :erl_eval.exprs(
+          parsed,
+          erl_binding
+        )
+
+      binding =
+        new_erl_binding
+        |> Map.drop(Map.keys(erl_binding))
+        |> Enum.reduce(binding, fn {name, value}, binding ->
+          binding
+          |> Keyword.put(
+            name
+            |> :erlang.atom_to_binary()
+            |> Macro.underscore()
+            |> :erlang.binary_to_atom(),
+            value
+          )
+        end)
+
+      {:ok, result, binding, env}
+    catch
+      kind, error ->
+        stacktrace = prune_stacktrace(__STACKTRACE__)
+
+        code_error =
+          if code_error?(error) and (error.file == env.file and error.file != "nofile") do
```

Comment:
I'm pretty sure Erlang won't raise any of the Elixir exceptions this function checks for, so it's effectively kind of a dead branch

Reply:
Good point. I left it in as a marker where to "translate" Erlang's errors.